tools/releaseBuild/azureDevOps/templates/release-ReleaseToNuGet.yml

```diff
@@ -30,7 +30,7 @@ steps:
     $releaseVersion = Get-Content "$ENV:PIPELINE_WORKSPACE/releasePipeline/metadata/release.json" | ConvertFrom-Json | Select-Object -ExpandProperty 'ReleaseVersion'
     $globalToolPath = "$ENV:PIPELINE_WORKSPACE/releasePipeline/finalResults/PowerShell.$releaseVersion.nupkg"
     ### -WhatIf to make sure we do not release global tool. Remove -WhatIf when the PowerShell name reservation is done.
-    Copy-Item $globalToolPath -Destination "$(Pipeline.Workspace)/release" -WhatIf
+    ### Copy-Item $globalToolPath -Destination "$(Pipeline.Workspace)/release" -WhatIf
```

Comment:
Shall this change be taken back to GH master?

Reply:
Seems like it shouldn't be -- I'll remove it